### ISSUE TITLE
Missing translations in Javasript Adapter

### ISSUE THREAD

**Issue by Feuer-sturm** [ioBroker/ioBroker.javascript#845]
Javascript Adapter 5.1.3

In Javascript Adapter some strings are not translated yet.

Following informations did I find in the browser console during surfing through Menu "Skripte".
For example:
![grafik](https://user-images.githubusercontent.com/25422320/119235584-7d9b7b80-bb33-11eb-9e4b-25f4b81be7a2.png)

```
Translate: ra_Deselect all
Translate: ra_Select all
Translate: ra_On weekdays
Translate: debug_help
Translate: debug
Translate: verbose_help
Translate: verbose
Translate: Debug instance
Translate: debug_help
Translate: easy
Translate: for programmers
Translate: for professionals
Translate: ra_Folders always first

Translate: Debug instance
Translate: Resume execution
Translate: Go to next line
Translate: Step into function
Translate: Step out from function
Translate: Stop on exception
Translate: Console
Translate: Stack
```

**Comment by Feuer-sturm** [comment]
Above mentions strings seems to be fixed in Javascript Adapter 5.2.8
I can not reproduce them and the buttons in cron tab are ok


Open Debug Instanz shows following missing translations:
```
Translate: Start
Translate: Filter
Translate: Instances
```

**Comment by GermanBluefox** [comment]
Fixed in 5.2.9

**Comment by Feuer-sturm** [comment]
fixed with 5.2.9
